proto/generate.proto

```diff
@@ -224,13 +224,17 @@ enum AdapterSource {
     S3 = 1;
     /// Adapters loaded via local filesystem path
     LOCAL = 2;
+    /// Adapters loaded via predibase    
+    PBASE = 3;
 }
 
 message DownloadAdapterRequest {
     /// Adapter ID
     string adapter_id = 1;
     /// Adapter source
     AdapterSource adapter_source = 2;
+    /// API Token for predibase
+    optional string predibase_api_token = 3;
```

Comment:
Can we rename to just `api_token`? I could see this being useful for HF hub token as well.

Reply:
ok will rename